site/docs/getting-started/DesignCards.tsx

```diff
@@ -11,42 +11,45 @@ type CardInfoType = { img: string; altText: string; content?: JSX.Element };
 const cardInfo: CardInfoType[] = [
   {
     img: Assets1,
-    altText: "figma components panel",
+    altText:
+      'Screenshot of Figma\'s assets panel displaying the "Salt: Components Light" library',
```

Comment:
I've heard two opinions on this phrasing.

1. Image of, Photo of etc. should not be used in alt text.
2. Screenshot of should be used for step by step instructions.

2. Seems to be coming mainly from educational (unis) sites.

Probably worth asking @maj-stella

Reply:
@joshwooding I've raised #823 for this